.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    reviewers:
+      - sondrelg
+
+  - package-ecosystem: pip
+    directory: /
+    versioning-strategy: lockfile-only
```

Comment:
NIP: from my experience, it's nice to have an `auto` versioning strategy when developing and maintaining a package/lib, because it's nice to have some info when a major version of any deps is updated, so we can add support for it in our package